Moxie KB/README.md

```diff
@@ -16,12 +19,12 @@ You will need to make the four follwing API calls. If you need to test them, you
 * Get Article – one call per document
 * Get Article Metadata – one call per document
 
-**Important:** There are two endpoints that carries same parameters, if you modify/reuse it don't forget to update both endpoints parameters the same way. The two parameters are NewlyPublished (for new articles), and RecentlyUpdated (for new versions of existing articles). When an article is updated with a new version, its record will cease to be part of the NewlyPublished list.
-    
+> **_IMPORTANT:_** There are two endpoints that carry the same parameters. If you modify/reuse endpoints, don't forget to update both endpoint parameters the same way. The two parameters are `NewlyPublished` (for new articles), and `RecentlyUpdated` (for new versions of existing articles). When an article is updated with a new version, its record will cease to be part of the NewlyPublished list.
```

Comment:
Should we use that markdown instead? It would make it standout a bit more. 
https://github.com/orgs/community/discussions/16925

Reply:
Done.